docs/using-the-compiler.rst

```diff
@@ -635,116 +635,88 @@ If you found a bug or if you have a feature request, please
 Example
 ~~~~~~~
 
-Assume you have the following contracts you want to update declared in ``Source.sol``:
+Assume that you have the following contract in ``Source.sol``:
 
-.. code-block:: none
+.. code-block:: solidity
 
-    // This will not compile after 0.5.0
+    pragma solidity >=0.6.0 <0.6.4;
+    // This will not compile after 0.7.0
     // SPDX-License-Identifier: GPL-3.0
-    pragma solidity >0.4.23 <0.5.0;
-
-    contract Updateable {
-        function run() public view returns (bool);
-        function update() public;
+    contract C {
+        // FIXME: remove constructor visibility and make the contract abstract
+        constructor() internal {}
     }
 
-    contract Upgradable {
-        function run() public view returns (bool);
-        function upgrade();
+    contract D {
+        uint time;
+
+        function f() public payable {
+            // FIXME: change now to block.timestamp
+            time = now;
+        }
     }
 
-    contract Source is Updateable, Upgradable {
-        function Source() public {}
+    contract E {
+        D d;
 
-        function run()
-            public
-            view
-            returns (bool) {}
+        // FIXME: remove constructor visibility
+        constructor() public {}
 
-        function update() {}
-        function upgrade() {}
+        function g() public {
+            // FIXME: change .value(5) =>  {value: 5}
+            d.f.value(5)();
+        }
     }
 
 
+
 Required changes
 ^^^^^^^^^^^^^^^^
 
-To bring the contracts up to date with the current Solidity version, the
-following upgrade modules have to be executed: ``constructor``,
-``visibility``, ``abstract``, ``override`` and ``virtual``. Please read the
-documentation on :ref:`available modules <upgrade-modules>` for further details.
+The above contract will not compile after 0.7.0. To bring the contract up to date with the current
```

Comment:
The wording is a bit confusing "after 0.7.0" makes me think of "0.8.0". I suggest using "with" or "starting with".

"will no longer compile starting with 0.7.0"

Reply:
Any change to this?